src/app/api/auth.api.model.ts

```diff
@@ -16,6 +16,11 @@ export interface GithubLoginCredentials {
     code: string;
 }
 
+export interface Web3WalletCredentials {
+    message: string;
+    signature: string;
+}
```

Comment:
It's not credentials. In the Web3 world, for a wallet (read account), credentials is a key pair (or seed phrase and wallet address)

Proposed naming: `Web3WalletOwnershipVerificationRequest`

Reply:
Fixed